pkg/plugin/plugin.go

```diff
@@ -120,5 +120,9 @@ func (p *plugin) CreateOrUpdate(ctx context.Context, cs *api.OpenShiftManagedClu
 	}
 
 	log.Info("starting health check")
-	return p.clusterUpgrader.HealthCheck(ctx, cs)
+	err = p.clusterUpgrader.HealthCheck(ctx, cs)
+	if err != nil {
+		return &api.PluginError{Err: err, Step: api.PluginStepWaitForConsoleHealth}
```

Comment:
It seems you have updated all simpleUpgrader methods apart from HealthCheck to return PluginError. Can we return this error from inside HealthCheck to stay consistent with the rest?

Reply:
done